tests/librispeech/eval.mk

```
@@ -0,0 +1,39 @@
+PYTHON = ./venv/bin/python3
```

It would be better to call a generic `python` command:

```suggestion
PYTHON = python
```

For example, I use `pyenv` to manage my python environments and this is not compatible with that workflow, while simply using `python` I believe it should work in both cases.